### ISSUE TITLE
Support http over native interface

### ISSUE THREAD

**Issue by gingerwizard** [ClickHouse/clickhouse-go#674]
HTTP can currently only be used over the standard interface. Implement support over native.

**Comment by r--w** [comment]
Yes, we are using CHProxy, thank you for your response. We will investigate that first.

**Comment by r--w** [comment]
@hileef you are right, after we updated CHProxy to the latest version the problem was gone. Thank you, guys 👍
